doc/source/user_guide/categorical.rst

```diff
@@ -625,6 +625,7 @@ even if some categories are not present in the data:
 ``DataFrame`` methods like :meth:`DataFrame.sum` also show "unused" categories.
 
 .. ipython:: python
+    :okwarning:
```

Comment:
can you change this so its a regular group op

Reply:
Done